### PR TITLE
Completed Day1 - Please review

### PR DESCRIPTION
**Pull request by bharatreddypothireddy**
Used Internal stack for push,pop,isFull,isEmpty functions . Implemented getMin() in O(1) time using an extra stack .

### CODE REVIEW

File: NewHashMap.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+    public static void main(String[] args) {
+        NewHashMap hm = new NewHashMap();
+        hm.put(3,33);
+        System.out.println(hm.get(3));
+        hm.remove(3);
+        System.out.println(hm.get(3));
+
+    }
+}
```
Comment (ashmeetdeol19) [inline NewHashMap.java]: This seems to be more like a brute force solution.

Comment (bharatreddypothireddy) [reply NewHashMap.java]: Yes , Will try to implement using linked lists .

**Comment by bharatreddypothireddy** [comment]
Implemented using ListNodes

File: NewHashMap.java (hunk 1 above)
Comment (bharatreddypothireddy) [reply NewHashMap.java]: Implemented

**Pull request closed without merging**